tool/data/helloworld.yaml

```diff
@@ -15,8 +15,18 @@ hook_helloworld:
 add_new_field:
   # Where we want to inject the hook code
   target: main
+  # What type of struct we want to inject the field into
   struct: MyStruct
+  # Which field we want to inject the code into
```

Comment:
Am I correct that at the moment to add more than one field we have to duplicate `struct rule`?

Reply:
Yes, but I agree that allowing a rule to apply to multiple fields would be a better approach.